### ISSUE TITLE
Consider adding a close button for mobile menu to improve user experience and usability

### ISSUE THREAD

**Issue by yannickiki** [WordPress/twentytwenty#625]
Currently on mobile, to open the menu the user has to click on the menu button. To close it, the user has to click on the same button. Nothing clearly indicates that clicking on the same button will close the menu.

![modal-mobile](https://user-images.githubusercontent.com/54663751/65814956-3e474b00-e1e9-11e9-8d36-d3df2e175b4c.jpg)

On desktop, the user experience is different, to close the menu there is a “Close Menu” label with a close icon. There is no ambiguity.

![modal-desktop](https://user-images.githubusercontent.com/54663751/65814839-81082380-e1e7-11e9-8ac2-f9352f40c967.jpg)

I suggest for a better user’s understanding and experience to show the Close Nav Toogle button on mobile and open the modal on top of the header. 

For design clarity, I don’t think it’s necessary to show the header (logo + search icon) when the modal is opened. Keeping the branding isn’t needed while browsing the menu.

![modal-mobile-proposition](https://user-images.githubusercontent.com/54663751/65814812-27075e00-e1e7-11e9-9ffc-25b9d190ee4b.jpg)

Here is a comparison:

![comparison](https://user-images.githubusercontent.com/54663751/65814961-48694980-e1e9-11e9-9dba-bf654a30c816.jpg)

**Comment by carolinan** [comment]
Please note that this is already being discussed here:
https://github.com/WordPress/twentytwenty/issues/80

**Comment by andersnoren** [comment]
@yannickiki You beat me to it 🙂 We've talked a bit about this in the last couple of days, and after the input from the accessibility people in the issue @carolinan linked to (with input from @afercia and @samikeijonen), we've decided to go this route (with the close button being displayed on all screen sizes).

In addition to making the modal more consistent and clear across all screen sizes, like you say, it will simplify the focus loop for the modal menu. It will also prevent issues with really long site titles or big logos causing the mobile menu to overlap with the header on mobile. It'll be easier all around. I'll submit a PR with this update during the day.
